svg/attributes/presentation.json

```diff
@@ -1505,6 +1505,39 @@
             }
           }
         },
+        "shape-subtract": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Attribute/shape-subtract",
+            "spec_url": "https://svgwg.org/svg2-draft/text.html#TextShapeSubtract",
+            "support": {
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": null
+              },
+              "ie": {
+                "version_added": null
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
```

Comment:
Sorry it took me some time to get back to your review here. I really appreciate your time and will add in those fixes.
But if we would rather not add in data for an unsupported feature that totally makes sense and we can close this out. Let me know what you think would be best, but otherwise again, thanks for your time.

I'll commit your suggested changes now and update it in a moment with the other relevant fixes, thanks!